### ISSUE TITLE
flysky i6

### ISSUE THREAD

**Issue by irsaadinugraha** [goebish/nrf24_multipro#30]
how to change the protocol for flysky i6...
need the ino file

**Comment by goebish** [comment]
Hi, there's nothing particular to do for the i6.
Just set channel order to TAER on the transmitter, or change order to AETR in the code.

**Comment by irsaadinugraha** [comment]
ok i will try thanks

**Comment by irsaadinugraha** [comment]
i try to change the code to AETR but not work...i still learning the command...can you make the code for flysky with AETR

**Comment by goebish** [comment]
You only have to move THROTTLE, just after ELEVATOR, in nrf24_multipro.ino, that's all ...

AILERON,
ELEVATOR,
THROTTLE,
RUDDER

**Comment by irsaadinugraha** [comment]
yes i do that but...but when finish i try to bind to eachine e011 still not bind...the light of arduino work find when binding...but not binding to my eachine e011

**Comment by irsaadinugraha** [comment]
i'm using nrf24l01+PA and arduino mini pro..

**Comment by goebish** [comment]
Check your wiring, nothing special is required in the code for the i6, many people got the module to work with it already.
http://carrerasfpv.com/forums/topic/2986/modulo-goebish-para-volar-los-minis-en-fly-sky-fs-i6/view/post_id/19103

**Comment by irsaadinugraha** [comment]
ok thanks

**Comment by noer96** [comment]
What is work to futaba t6j?

**Comment by goebish** [comment]
Yes, it works with Futaba transmitters.

**Comment by noer96** [comment]
Maybe I need you Help,  because I beginer for coding

Pada tanggal 27 Jan 2018 5.02 PM, "goebish" <notifications@github.com>
menulis:
